src/controllers/transaction/routes.ts

```diff
@@ -0,0 +1,52 @@
+import { Router } from "restify-router";
+import * as errors from "restify-errors";
+import logger = require("../../utils/logger");
+import { TransactionService } from "./service";
+import { DefaultHTTPHandler } from "../httpHandler";
+import winston = require("winston");
+import { Next, Response } from "restify";
+import { RequestWithContext } from "../../types/restify";
+
+export class TransactionHTTPHandler extends DefaultHTTPHandler {
+  transactionService: TransactionService;
+
+  constructor(logger: winston.Logger, transactionService: TransactionService) {
+    super(logger);
+    this.transactionService = transactionService;
+  }
+
+  public SetupRoutes(): Router {
+    const TransactionRouter = new Router();
+    TransactionRouter.get(
+      "/ping",
+      /*** @todo: put AuthMiddleware here ***/
```

Comment:
Sorry bro. It seems like we are going to have to take in the Auth middleware from the user service. Just pass the whole service there (I believe its already there)